api-spec.md

```diff
@@ -6,7 +6,139 @@ The STAC API defines a RESTful JSON-based server to query [SpatioTemporal Asset
+| **`/collections/{collectionId}`**               | collection         | Collection     | Returns single Collection JSON |
+| **`/collections/{collectionId}/items`**         | items                   | ItemCollection | GeoJSON FeatureCollection-conformant entity of Items in collection |
+| **`/collections/{collectionId}/items/{featureId}`** | Item           | Returns single Item (GeoJSON Feature) |
+
+### Conformance Classes
+
+STAC API is evolving to utilize OAFeat's 
+'[Conformance](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_declaration_of_conformance_classes)' JSON structure. For 
+STAC API 1.0.0-beta.1 we declare new STAC Conformance classes, and specify which OAFeat ones to use. These are detailed in the relevant
+sections below.The core STAC conformance classes communicate the conformance JSON only in the root (`/`) document, while OGC API 
+requires they also live at the `/conformance` endpoint. 
+
+**NOTE:** *By 1.0.0 we aim to have requirements classes specified in detail, as testable assertions, 
+like OGC does, but for now the core reference is just this spec document and the OpenAPI yaml. We also desire to have the
+URI's for conformance to actually resolve to machine-readable information clients can use.*
+
+**Work In Progress**: The below is still in flux - with just the headings fleshed out.
+
+## STAC Core and OAFeat Versions
+
+## STAC API 
+
+### STAC API Core
+
+The core of a STAC API is its landing page, which is the starting point to discover STAC data and what the API supports.
+
+```json
+{
+    "stac_version": "1.0.0-beta.2",
+    "id": "example-stac",
+    "title": "A simple STAC API Example",
+    "description": "This Catalog aims to demonstrate the a simple landing page",
+    ""
+    "links": [
+        {
+            "rel": "child",
+            "href": "https://stacserver.org/collections/sentinel-2",
+        },
+        {
+            "rel": "child",
+            "href": "https://stacserver.org/collections/landsat-8",
+        },
+        {
+            "rel": "search",
+            "type": "application/json",
+            "href": "https://stacserver.org/search"
+        }
+    ],
+    "conformsTo" : [
+        "http://stacspec.org/spec/api/1.0.0-beta.1/req/stac-api-core",
+        "http://stacspec.org/spec/api/1.0.0-beta.1/req/stac-search"
+    ]
+}
+```
+
+There are a few things to note: 
+
+- The returned JSON is a valid [STAC Catalog](stac-spec/catalog-spec/catalog-spec.md), and it can provide any number of 'child' links
+to navigate down to additional Catalogs, Collections & Items.
+- The `links` section is a required part of STAC Catalog, and serves as the list of API endpoints. These can live at any location, the 
+client must inspect the the `rel` to understand what capabilities are offered at each location.
+- The `conformsTo` section follows exactly the structure of OGC API - Features [declaration of conformance 
```

Comment:
I'm not completely sold on breaking from OGC API, how many users are really going to use the conformance classes enough so they should be at landing page?  Don't the provided links indicate what capabilities are provided?  I'm not sure it's worth doing something different.

Reply:
I can still go either way. Was compelled by @m-mohr's arguments, I think the main ones are https://github.com/opengeospatial/oapi_common/issues/71 I tend to think about being nicer to clients, particularly javascript ones, and saving an extra request just to get conformance seems nice. I wouldn't say it's 'breaking' with OGC API, it's just a nice shortcut if OGC API is implemented, and if it's not implemented then it's a straightforward path.

Reply:
Oh, and the point to me isn't users - it's programmatic clients, that read the conformance classes to figure out the capabilities. Like can it make a 'sort' or 'fields' query, or does it have to request everything and do sort / fields on the client side? Is CQL supported? Etc. I think most API clients will want to know what extensions are available, so let them do it with their first request.